docs/development/core/public/kibana-plugin-public.basepathsetup.addtopath.md

```diff
@@ -0,0 +1,22 @@
+[Home](./index) &gt; [kibana-plugin-public](./kibana-plugin-public.md) &gt; [BasePathSetup](./kibana-plugin-public.basepathsetup.md) &gt; [addToPath](./kibana-plugin-public.basepathsetup.addtopath.md)
+
+## BasePathSetup.addToPath() method
+
+Add the current basePath to a path string.
+
+<b>Signature:</b>
+
+```typescript
+addToPath(path: string): string;
+```
+
+## Parameters
+
+|  Parameter | Type | Description |
+|  --- | --- | --- |
+|  path | <code>string</code> | A relative url including the leading <code>/</code>, otherwise it will be returned without modification |
+
+<b>Returns:</b>
+
+`string`
+

```

Comment:
nit: Remove this extra trailing line.

Reply:
These docs are generated by api-documenter (I just bumped the version to include a PR I submitted to add a "do not edit header" so that this is clearer). At some point we might fork api-documenter to customize the formatting to our needs since I agree that the formatting like 4-space indentation is different from our standards.